### PR TITLE
Copyedits

### PR DESCRIPTION
**Pull request by Kenadia**
I think the page is a little too wordy, and we could do better to make CarlHacks look awesome by emphasizing the parts that are going to be awesome. Other concerns: people familiar with hackathons will be looking for only a few things: reimbursement for flights, sponsors, prizes, deadlines—which are not easy to find right now without reading several paragraphs of text. More photos would be cool but not easy since it would require reconsidering the design.

**my changes**: Nothing major, but tried to cut down words without cutting content. If the diffs are hard to read, check out this screenshot for a side-by-side comparison of the Event Details https://www.dropbox.com/s/xaxzfokw3x71bec/Screenshot%202015-03-08%2017.15.47.png?dl=0

### CODE REVIEW

**Comment by lord** [comment]
:+1: All these changes look great to me.

**Comment by lord** [comment]
Trying to think of a way to summarize the information faster for seasoned hackathoners. Maybe a "quick facts" sidebar at the top?

**Comment by Kenadia** [comment]
Or a section titled “Why carlhacks?”
- Priority registration option for women
- Bouldering cave and 440-acre arboretum on site
- Travel reimbursements
- First MLH-sponsored hackathon at a liberal arts college (**is this true?**)
- We're a passionate group dedicated to making this an awesome experience for all (link to photos)

**Comment by lord** [comment]
I like that idea! I'm going to merge this now. Did you want to write that section? If not, I'd be glad to.

**Comment by Machyne** [comment]
LGTM!
